metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -556,7 +644,39 @@ func (sink *StackdriverSink) getResourceLabels(labels map[string]string) map[str
 	}
 }
 
-func createTimeSeries(resourceLabels map[string]string, metadata *metricMetadata, point *sd_api.Point) *sd_api.TimeSeries {
+func (sink *StackdriverSink) getContainerResourceLabels(labels map[string]string) map[string]string {
+	return map[string]string{
+		"project_id":     sink.project,
+		"location":       sink.zone,
```

Comment:
Add todo to revisit how location is set.

Currently it's based on the zone where Heapster is running and it could be something else because we run nodes in multiple zone.